authenticatorbridge/src/test/java/com/bitwarden/authenticatorbridge/manager/AuthenticatorBridgeManagerTest.kt

```diff
@@ -0,0 +1,440 @@
+
+        assertEquals(AccountSyncState.Loading, manager.accountSyncStateFlow.value)
+        verify { context.bindService(any(), any(), Context.BIND_AUTO_CREATE) }
+    }
+
+    @Test
+    @Suppress("MaxLineLength")
+    fun `onServiceConnected when symmetric key is not present and service returns null symmetric key state should be SyncNotEnabled and should unbind service`() {
+        val serviceConnection = slot<ServiceConnection>()
+        val mockIntent: Intent = mockk()
+        fakeSymmetricKeyStorageProvider.symmetricKey = null
+        every { mockBridgeService.symmetricEncryptionKeyData } returns null
+        every { IAuthenticatorBridgeService.Stub.asInterface(any()) } returns mockBridgeService
+        every {
+            anyConstructed<Intent>().setComponent(any())
+        } returns mockIntent
+        every { context.unbindService(any()) } just runs
+        every {
+            context.bindService(
+                any(),
+                capture(serviceConnection),
+                Context.BIND_AUTO_CREATE
+            )
+        } returns true
+
+        fakeLifecycleOwner.lifecycle.dispatchOnStart()
+        serviceConnection.captured.onServiceConnected(mockk(), mockk())
+
+        assertEquals(AccountSyncState.SyncNotEnabled, manager.accountSyncStateFlow.value)
+        verify { mockBridgeService.symmetricEncryptionKeyData }
+        verify { context.bindService(any(), any(), Context.BIND_AUTO_CREATE) }
+        verify { context.unbindService(any()) }
+    }
+
+    @Test
+    @Suppress("MaxLineLength")
+    fun `onServiceConnected when symmetric key does not match should set symmetric key from service`() {
+        fakeSymmetricKeyStorageProvider.symmetricKey = null
+        every { mockBridgeService.symmetricEncryptionKeyData } returns SYMMETRIC_KEY
+        val serviceConnection = slot<ServiceConnection>()
+        val mockIntent: Intent = mockk()
+        every { IAuthenticatorBridgeService.Stub.asInterface(any()) } returns mockBridgeService
+        every {
+            anyConstructed<Intent>().setComponent(any())
+        } returns mockIntent
+        every { context.unbindService(any()) } just runs
+        every {
+            context.bindService(
+                any(),
+                capture(serviceConnection),
+                Context.BIND_AUTO_CREATE
+            )
+        } returns true
+
+        fakeLifecycleOwner.lifecycle.dispatchOnStart()
+        serviceConnection.captured.onServiceConnected(mockk(), mockk())
+
+        assertEquals(AccountSyncState.Loading, manager.accountSyncStateFlow.value)
+        assertEquals(fakeSymmetricKeyStorageProvider.symmetricKey, SYMMETRIC_KEY)
+        verify { mockBridgeService.symmetricEncryptionKeyData }
+        verify { context.bindService(any(), any(), Context.BIND_AUTO_CREATE) }
+    }
+
+    @Test
+    @Suppress("MaxLineLength")
+    fun `onServiceConnected when symmetric key does match not query for symmetric key`() {
```

Comment:
Seems like something is missing in this one.

Reply:
Yep, the word "should": 2680fd8f95f194121723ab94bfedc389de4468b6